vector/src/main/java/im/vector/app/features/location/LocationSharingViewState.kt

```diff
@@ -43,7 +44,8 @@ data class LocationSharingViewState(
 
 fun LocationSharingViewState.toMapState() = MapState(
         zoomOnlyOnce = true,
-        pinLocationData = lastKnownLocation,
+        userLocationData = lastKnownUserLocation,
         pinId = DEFAULT_PIN_ID,
-        pinDrawable = pinDrawable
+        pinDrawable = null,
+        showPin = areTargetAndUserLocationEqual == false
```

Comment:
`.orFalse` can be used here too.

Reply:
Actually I replaced it by `areTargetAndUserLocationEqual.orTrue().not()` to be equivalent to `areTargetAndUserLocationEqual == false`.